components/usage/pkg/stripe/stripe.go

```diff
@@ -320,6 +320,62 @@ func (c *Client) GetSubscriptionWithCustomer(ctx context.Context, subscriptionID
 	return subscription, nil
 }
 
+func (c *Client) CreateSubscription(ctx context.Context, customerID string, priceID string) (*stripe.Subscription, error) {
+	if customerID == "" {
+		return nil, fmt.Errorf("no customerID specified")
+	}
+	if priceID == "" {
+		return nil, fmt.Errorf("no priceID specified")
+	}
+
+	params := &stripe.SubscriptionParams{
+		Customer: &customerID,
+		Items: []*stripe.SubscriptionItemsParams{
+			{
+				Price: &priceID,
+			},
+		},
+	}
+
+	subscription, error := c.sc.Subscriptions.New(params)
```

Comment:
`error` is a native Go type, here you're shadowing that type. (Unsure why the linter is not picking this up, will investigate).
The more idiomatic is to use `err` as the return variable name.